cockroachcloud/cluster-management.md

```diff
@@ -171,16 +171,14 @@ You can use the [**Databases** page](databases-page.html) to create a new databa
 
 ## Restore data from a backup
 
-Cockroach Labs runs full backups daily and incremental backups hourly for every {{ site.data.products.dedicated }} cluster. The full backups are retained for 30 days and incremental backups for 7 days.
+Cockroach Labs runs full backups daily and incremental backups hourly for every {{ site.data.products.dedicated }} cluster. The full backups are retained for 30 days and incremental backups for 7 days. Use the [Managed-Service Backups](use-managed-service-backups.html?filters=dedicated#ways-to-restore-data) to restore from automatic backups in the Console. 
```

Comment:
Here:

> Use the [Managed-Service Backups](use-managed-service-backups.html?filters=dedicated#ways-to-restore-data) to restore from automatic backups in the Console.

Was the intention to say: "Use the Managed Service Backups page to restore...."?

Perhaps:

> See the Use Managed-Service Backups page for ways to restore data from your cluster's automatic backups in the Console.